src/DeferredFutures.jl

```diff
@@ -13,6 +13,15 @@ else
 end
 
 """
+    show(io::IO, ref::RemoteChannel)
+
+Print a simplified string representation of the `RemoteChannel` with its parameters.
+"""
+function Base.show(io::IO, ref::RemoteChannel)
```

Comment:
Unfortunately you shouldn't do this. This overrides Base behaviour for Base types, which is called "type piracy". 

On the bright side, you are a pirate!